ddcz/forms/signup.py

```diff
@@ -66,35 +66,35 @@ class GdprChoices(Enum):
         ),
     )
 
-    jmeno = forms.CharField(
+    name_given = forms.CharField(
         label="",
         widget=forms.TextInput(
             attrs={"placeholder": "Křestní jméno", "id": "first_name"}
         ),
     )
 
-    prijmeni = forms.CharField(
+    name_first = forms.CharField(
```

Comment:
Is not the first name the same things as given name in english?
Like first - last or given - family?

Reply:
Thanks for the catch! Yes, given/family